client/src/main/java/io/pravega/client/control/impl/ControllerImpl.java

```diff
@@ -1508,6 +1511,31 @@ private TxnSegments convert(CreateTxnResponse response) {
                 });
     }
 
+    @Override
+    public AsyncIterator<io.pravega.controller.stream.api.grpc.v1.Controller.TxnResponse> listCompletedTransactions(Stream stream) {
+        Exceptions.checkNotClosed(closed.get(), this);
+        Preconditions.checkNotNull(stream, "stream");
+        final long requestId = requestIdGenerator.get();
+        long traceId = LoggerHelpers.traceEnter(log, "listCompletedTransactions", stream, requestId);
```

Comment:
`LIST_COMPLETED_TRANSACTIONS` can be used

Reply:
Done.